README.md

```diff
@@ -20,24 +20,23 @@ Lastly, install the webdrivers of the browsers you wish to use. A webdriver is a
 
 After installing, you can now run the `axe` command in your CLI, followed by the URL of the page you wish to test:
 
-	axe https://www.deque.com
+    axe https://www.deque.com
```

Comment:
These code examples should be highlighted in some way other than plain text. It looks like they were list items before, perhaps they should be code blocks instead?

Reply:
They're code snippets. Markdown treats text with 4 indents as code.

I'll refactor this and use the normal code fence instead, as this is a strange/not-well-known langauge feature.